app/api/routes.py

```
@@ -170,12 +184,47 @@ def patch_record_exchange(
 
+# 모든 복막투석기록 조회 api (공통 + 회차)
+@router.get(
+    "/api/v1/records",
+    tags=["복막투석기록"],
+    summary="환자의 모든 기록 조회",
+    description="특정 년도와 월에 해당하는 환자의 모든 투석기록을 조회합니다."
+)
+def get_records(
+        year: int = Query(2025, ge=2000, description="조회할 기록의 연도"),
+        month: int = Query(11, ge=1, le=12, description="조회할 기록의 월"),
+        db: Session = Depends(get_db),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)